source/clear-linux/guides/maintenance/distro-factory.rst

```diff
@@ -0,0 +1,109 @@
+.. distro_factory:
+
+Distro factory
+##############
+
+Distro factory gives DevOps teams the means to produce |CL-ATTR|
+derivatives via pipeline jobs that support a release workflow with
+continuous integration. This guide helps you deploy the infrastructure.
+
+.. contents::
+   :local:
+   :depth: 1
+
+Description
+***********
+
+As an operating system vendor (OSV) toolchain, Distro factory manages |CL|
+derivatives while capturing the requirements used to maintain the release
+cadence over time. Distro factory provides a wrapper around
+:ref:`mixer <mixer>` that enables a DevOps team to publish rolling releases
+that include the update content and metadata associated with each version.
+
+Maintaining an OSV derivative of |CL| requires:
+
+* Monitoring upstream |CL| for new releases
+* Building software packages and staging
+* Employing CI/CD automation for building releases
+* Integrating Quality Assurance for testing and validation
+
+How it works
+************
+
+Review the requirements and learn the basics of implementation.
+
+.. contents::
+   :local:
+   :depth: 1
+
+Prerequisites
+=============
+
+* A repository with software RPM artifacts and a CI/CD system with a |CL|
+  machine for building `mixes`
+
+* Experience using :ref:`mixer <mixer>` to create a |CL|-based distro
+
+* Experience using :ref:`swupd <swupd-guide>`
+
+Overview
+========
+
+Distro factory is divided in two parts: *Content Workflow*; and *Release
```

Comment:
In fact, the maintenance process of Clear Linux is divided in those two parts. Bistro-Factory only handles the 'Release Workflow'. We only mention the Content Workflow as an explanation of the entire process.

Reply:
@mbelluzzo. I will rephrase the explanation per your comment. Thanks.